<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>groupemodule</class>
 <widget class="QWidget" name="groupemodule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="groupemodule_menu">
    <widget class="QPushButton" name="menu">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>490</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Main Menu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="update">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Update groupemodule</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>671</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 22pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 0);</string>
     </property>
     <property name="text">
      <string>                  groupe module Managment </string>
     </property>
    </widget>
    <widget class="QPushButton" name="show">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>400</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Show groupemodule</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteS">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>320</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete groupemodule</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Adder">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Add groupemodule</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="add_groupemodule">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>540</y>
       <width>191</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>861</width>
       <height>521</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="IDGM">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_id">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>     Enter ID groupe module</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NameGM">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2_Name">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>140</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>      Enter Name of The groupe module</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="coefficientGM">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3_coef">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>   Enter  Coefficent of groupe module</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_idmat">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>130</y>
        <width>131</width>
        <height>391</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID mat</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="idmmat">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="save">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>330</y>
        <width>81</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3_coef_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>300</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter ID of The Subject</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>671</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 22pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>                Add  groupe module </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushB">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="show_groupemodule">
    <widget class="QLineEdit" name="searchField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Search For Student</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>-20</y>
       <width>681</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 22pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>                Liste of Groupe Modulde </string>
     </property>
    </widget>
    <widget class="QTableView" name="groupemodule_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>421</width>
       <height>381</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Goback">
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>510</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Go Back</string>
     </property>
    </widget>
    <widget class="QTableView" name="groupemodule_matiere">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>120</y>
       <width>441</width>
       <height>381</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="update_groupemodule">
    <widget class="QPushButton" name="updateC">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>310</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>update</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idGM_up">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>80</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="NewMat">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>450</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter new matiere :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updateN">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>240</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>update</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter the ID of the groupe module </string>
     </property>
    </widget>
    <widget class="QPushButton" name="Name_GM">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>240</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updateM">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>440</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_GM">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="coef_P_GM">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>310</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="coef_GM">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>310</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>coef</string>
     </property>
    </widget>
    <widget class="QLabel" name="newCoef">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>280</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter new coefficent:</string>
     </property>
    </widget>
    <widget class="QLabel" name="newName">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>210</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>enter new Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Name_GM_P">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>240</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="mat_GM">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>380</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>subject</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_GM_update">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>261</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="GoB">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>490</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Go back</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_GMM_update">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>251</width>
       <height>271</height>
      </rect>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QComboBox" name="mat_GM_P">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>440</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="oldIdmat">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>380</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="old_idmat">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>350</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter the ID mat to update :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="delete_groupemodule">
    <widget class="QLineEdit" name="coef">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>320</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nomGM">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>250</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_GM">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>390</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushbotton{ }
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(170, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>DELETE</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableGM_del">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>581</height>
      </rect>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLineEdit" name="ID_GM">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>180</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="go_b_del">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>0</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
